src/jit/gentree.h

```diff
@@ -4009,7 +4016,7 @@ struct GenTreeCopyOrReload : public GenTreeUnOp
             return gtRegNum;
         }
 
-#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
+#if FEATURE_MULTIREG_RET
```

Comment:
In gentree.cpp there is another function GenTreeCall::GetOtherRegMask() that is implemented only for FEATURE_UNIX_AMD64_STRUCT_PASSING and that should be changed to FEATURE_MULTIREG_RET